src/templates/account.html

```diff
@@ -3,29 +3,31 @@
 
 {% block content %}
 
-    <!-- Begin Page Content -->
-    <div class="container-fluid">
+<!-- Begin Page Content -->
+<div class="container-fluid">
 
-        <!-- Page Heading -->
-        <div class="d-sm-flex align-items-center justify-content-between mb-4">
-            <h1 class="h3 mb-0 text-gray-800">Account</h1>
-        </div>
-
-        <!-- Content Row -->
-        <div class="row">
-            <div class="col">
-                You are currently logged in as {{ name }}
-                {% if not current_user.active %}
-                , however your account is not active yet. 
-                <p>Therefore you cannot access the data. For more information contact bioinformatica-genetica@umcutrecht.nl
+    <!-- Page Heading -->
+    <div class="d-sm-flex align-items-center justify-content-between mb-4">
+        <h1 class="h3 mb-0 text-gray-800">Account</h1>
+    </div>
 
-                <p>
-                <a class="btn btn-primary" href="{{ url_for('logout') }}">logout</a> <a class="btn btn-default" href="{{ url_for('register') }}">register</a>
-                </p>
-                {% endif %}
-            </div>
+    <!-- Content Row -->
+    <div class="row">
+        <div class="col">
+            You are currently logged in as {{ name }},
```

Comment:
This will always display a ','.

Reply:
but normally you are never in the "account" page, if you are not logged in (unless you type it directly in the web adres). 
The whole sentence will always be visible, not just the ",".
What is your suggestion here?

Reply:
I see what you mean now
```
Account
You are currently logged in as 123_diaggen@123.com,
```
I will just remove the ",". is not really necessary